recipes/oneloop/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "oneloop" %}
+{% set version = "3.7.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/hameren/{{ name }}/get/v{{ version }}.tar.gz
+  sha256: e03ed0e852f360fb19803dc6aaf83812604059ff9ae6baac5d856dbdf9b8797d
+
+build:
+  skip: true  # [win]
```

Comment:
This is currently failing on macOS as all conda-forged/staged-recipes are failing on macOS at the moment (2024-11-15).